lib/livebook_web/live/session_live/external_window_live.ex

```diff
@@ -0,0 +1,146 @@
+defmodule LivebookWeb.SessionLive.ExternalWindowLive do
+  use LivebookWeb, :live_view
+
+  alias Livebook.{Notebook, Session, Sessions}
+  alias Livebook.Notebook.Cell
+
+  @impl true
+  def mount(%{"id" => session_id, "type" => type} = params, _session, socket) do
+    case Sessions.fetch_session(session_id) do
+      {:ok, %{pid: session_pid}} ->
+        {data, client_id} =
+          if connected?(socket) do
+            {data, client_id} =
+              Session.register_client(session_pid, self(), socket.assigns.current_user)
+
+            Session.subscribe(session_id)
+
+            {data, client_id}
+          else
+            data = Session.get_data(session_pid)
+            {data, nil}
+          end
+
+        session = Session.get_by_pid(session_pid)
+
+        {:ok,
+         socket
+         |> assign(
+           session: session,
+           client_id: client_id,
+           type: type,
+           embedded?: params["embedded"] == "true",
+           data_view: data_to_view(data)
+         )
+         |> assign_private(data: data)}
+
+      :error ->
+        # TODO: handle this error correctly
+        {:ok, redirect(socket, to: ~p"/")}
+    end
+  end
+
+  @impl true
+  def mount(%{"id" => session_id}, _session, socket) do
+    {:ok, redirect(socket, to: ~p"/sessions/#{session_id}")}
+  end
+
+  defp assign_private(socket, assigns) do
+    Enum.reduce(assigns, socket, fn {key, value}, socket ->
+      put_in(socket.private[key], value)
+    end)
+  end
+
+  defp data_to_view(data) do
+    %{
+      notebook_name: data.notebook.name,
+      output_view: enrich_output_panel_data(data)
+    }
+  end
+
+  @impl true
+  def render(%{type: "output-panel"} = assigns) do
+    ~H"""
+    <div
+      id="external-window"
+      phx-hook="ExternalWindow"
+      data-window-embedded={@embedded?}
+      data-el-output-panel
+    >
+      <div
+        data-el-js-view-iframes
+        class="pointer-events-none"
```

Comment:
did this for now, so hover events on item are triggered for the drag panel.

Reply:
Fixed in [34e283a](https://github.com/livebook-dev/livebook/pull/2109/commits/34e283a43d2fa7f2b6dddae577288d6569aa1a78)